pkg/metrics/commonMetrics.go

```diff
@@ -21,33 +20,33 @@ import (
 	"reflect"
 )
 
-var DEFAULT_BUCKETS = []float64{0.000_25, 0.000_5, 0.001, 0.005, 0.010, 0.015, 0.025, 0.050, 0.100, 0.250, 0.500, 1.0}
+var DefaultBuckets = []float64{0.000_25, 0.000_5, 0.001, 0.005, 0.010, 0.015, 0.025, 0.050, 0.100, 0.250, 0.500, 1.0}
 
 const (
-	DEFAULT_SESSION_ALIAS_LABEL_NAME = "session_alias"
-	DEFAULT_DIRECTION_LABEL_NAME     = "direction"
-	DEFAULT_EXCHANGE_LABEL_NAME      = "exchange"
-	DEFAULT_ROUTING_KEY_LABEL_NAME   = "routing_key"
-	DEFAULT_QUEUE_LABEL_NAME         = "queue"
-	DEFAULT_MESSAGE_TYPE_LABEL_NAME  = "message_type"
-	DEFAULT_TH2_PIN_LABEL_NAME       = "th2_pin"
-	DEFAULT_TH2_TYPE_LABEL_NAME      = "th2_type"
-	MESSAGE_GROUP_TH2_TYPE           = "MESSAGE_GROUP"
-	EVENT_TH2_TYPE                   = "EVENT"
-	RAW_MESSAGE_TYPE                 = "RAW_MESSAGE"
-	MESSAGE_TYPE                     = "MESSAGE"
+	DefaultSessionAliasLabelName = "session_alias"
+	DefaultDirectionLabelName    = "direction"
+	DefaultExchangeLabelName     = "exchange"
+	DefaultRoutingKeyLabelName   = "routing_key"
+	DefaultQueueLabelName        = "queue"
+	DefaultMessageTypeLabelName  = "message_type"
+	DefaultTh2PinLabelName       = "th2_pin"
+	DefaultTh2TypeLabelName      = "th2_type"
+	MessageGroupTh2Type          = "MESSAGE_GROUP"
+	EventTh2Type                 = "EVENT"
+	RawMessageType               = "RAW_MESSAGE"
+	ParsedMessageType            = "MESSAGE"
```

Comment:
For mine information. What is the correct naming convention.
I found two opinions: 
https://www.golangprograms.com/go-language/constants.html
https://go.dev/doc/effective_go#mixed-caps

Reply:
I would believe the official guide from go.dev. Also, IDE for golang reports the original formatting